modules/move-etcd-different-disk.adoc

```diff
@@ -8,22 +8,36 @@
 
 You can move etcd from a shared disk to a separate disk to prevent or resolve performance issues.
 
-.Prerequisites
-
-*  The `MachineConfigPool` must match `metadata.labels[machineconfiguration.openshift.io/role]`. This applies to a controller, worker, or a custom pool.
-*  The node's auxiliary storage device, such as `/dev/sdb`, must match the sdb. Change this reference in all places in the file.
+The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
 [NOTE]
 ====
 This procedure does not move parts of the root file system, such as `/var/`, to another disk or partition on an installed node.
 ====
 
-The Machine Config Operator (MCO) is responsible for mounting a secondary disk for an {product-title} {product-version} container storage.
+.Prerequisites
 
-Use the following steps to move etcd to a different device:
+* You have installed the {oc-first}.
+* You have access to the cluster with `cluster-admin` privileges.
+* The `MachineConfigPool` must match `metadata.labels[machineconfiguration.openshift.io/role]`. This applies to a controller, worker, or a custom pool.
 
 .Procedure
-.  Create a `machineconfig` YAML file named `etcd-mc.yml` and add the following information:
+
+. Attach the new disk to the cluster and verify that the disk is detected in the node by using the `lsblk` command in a debug shell:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
++
+[source,terminal]
+----
+# lsblk
+----
++
+Note the device name of the new disk reported by the `lsblk` command.
+
+. Create a `MachineConfig` YAML file named `etcd-mc.yml` with contents such as the following, replacing `<new_disk_name>` with the noted device name:
```

Comment:
```suggestion
. Create a `MachineConfig` YAML file named `etcd-mc.yml` with contents such as the following, replacing instances of `<new_disk_name>` with the noted device name:
```

Reply:
Suggestion: just to clarify that there's more than one area in the YAML where `new_disk_name` needs replacing.